lib/semver.go

```diff
@@ -2,52 +2,37 @@ package lib
 
 import (
 	"fmt"
-	"sort"
 
 	semver "github.com/hashicorp/go-version"
 )
 
 // GetSemver : returns version that will be installed based on server constaint provided
```

Comment:
```suggestion
// GetSemver : returns version that will be installed based on server constraint provided
```

Reply:
Yep, I know. Appreciate all the someone else's bits you've got fixed in scope of this straightly say refactoring PR 🙇🏻‍♂️